src/main/webapp/cdn/blockly/generators/propc/console.js

```diff
@@ -122,14 +122,17 @@ Blockly.Blocks.console_move_to_row = {
 
 Blockly.propc.console_newline = function () {
     return 'print("\\r");';
+    Blockly.propc.serial_terminal_ = true;
```

Comment:
Thanks, I'll fix it in the morning when I get back online.

On Aug 28, 2016 1:31 AM, "michel-cf" <notifications@github.com> wrote:

> In src/main/webapp/cdn/blockly/generators/propc/console.js
> <https://github.com/parallaxinc/BlocklyProp/pull/617#discussion_r76528231>
> :
>
> > @@ -122,14 +122,17 @@ Blockly.Blocks.console_move_to_row = {
> >
> >  Blockly.propc.console_newline = function () {
> >      return 'print("\\r");';
> > +    Blockly.propc.serial_terminal_ = true;
>
> This won't get executed as it is below the return statement. ( same for
> console_clear)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/parallaxinc/BlocklyProp/pull/617/files/a14b7a8d669a8ae6f35995ca8bbcdd7969636d9d#r76528231>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AS0quCmDbxplgTOo6z9htLUqDXX7lyhrks5qkUdtgaJpZM4Ju2Bx>
> .
>